addons/project/static/src/components/project_task_name_with_subtask_count_char_field/project_task_name_with_subtask_count_char_field.js

```diff
@@ -6,7 +6,7 @@ import { formatChar } from '@web/views/fields/formatters';
 
 class ProjectTaskNameWithSubtaskCountCharField extends CharField {
     get formattedSubtaskCount() {
-        return formatChar(this.props.record.data.allow_subtasks && this.props.record.data.child_text || '');
+        return formatChar(this.props.record.data.subtask_count ? `(${this.props.record.data.closed_subtask_count}/${this.props.record.data.subtask_count} subtask${this.props.record.data.subtask_count>1 ? 's' : ''})`  : '');
```

Comment:
```suggestion
        return this.props.record.data.subtask_count
             ? `(${this.props.record.data.closed_subtask_count}/${this.props.record.data.subtask_count} subtasks)`
             : "";
```
Sorry I hadn't seen that before :see_no_evil: 
- Why use `formatChar`?
- Break the line.
- We don't handle singular/plural. (Think about other languages :nauseated_face: )
- Use "" i/o ''. (don't ask why, idk)

Reply:
i don't know why formatChar, it was there 
why breaking the line ? 
thanks for the info
why ? :p